docs/v0.9/en/getting-started/installation/greptimedb-cluster.md

```diff
@@ -1,73 +1,150 @@
 
 ```
-./gtctl cluster create mycluster
+helm repo add greptime https://greptimeteam.github.io/helm-charts/
+helm repo update
 ```
 
-After the creation is completed, you can use the following command to connect the cluster with MySQL protocol:
+You can run the following command to see the charts:
 
 ```
-./gtctl cluster connect mycluster
+helm search repo greptime
 ```
 
-You also can use [Helm Charts](/user-guide/operations/deploy-on-kubernetes/overview.md) to deploy the cluster.
+### Step 2: Deploy the etcd cluster
+
+We recommend using the Bitnami etcd [chart](https://github.com/bitnami/charts/blob/main/bitnami/etcd/README.md) to deploy the etcd cluster:
+
+```
+helm upgrade \
+  --install etcd oci://registry-1.docker.io/bitnamicharts/etcd \
+  --set replicaCount=3 \
+  --set auth.rbac.create=false \
+  --set auth.rbac.token.enabled=false \
+  --create-namespace \
+  -n etcd-cluster
+```
 
 :::tip Note
 
-You can use the [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create the Kubernetes:
+You can skip the step if you already have an available etcd cluster in Kubernetes.
+
+:::
+
+### Step 3: Deploy greptimedb-operator
+
+The [greptimedb-operator](https://github.com/GrepTimeTeam/greptimedb-operator) is a Kubernetes operator that manages the GreptimeDB cluster. We can install greptimedb-operator in the `greptimedb-admin` namespace:
 
+``` 
+helm upgrade \
+  --install \
+  --create-namespace \
+  greptimedb-operator greptime/greptimedb-operator \
+  -n greptimedb-admin
 ```
-kind create cluster
+
+### Step 4: Deploy the GreptimeDB cluster
+
+We can deploy the cluster by using the default storage in the `default` namespace:
+
+```
+helm upgrade \
+  --install mycluster \
+  --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
+  --set image.tag=<version>
+  greptime/greptimedb-cluster \
+  -n default
 ```
 
-:::
+### Step 5: Use the `kubectl port-forward` to access the cluster
 
-When the cluster is ready on your Kubernetes, you can use the following commands to expose all the service ports(make sure you already install [kubectl](https://kubernetes.io/docs/tasks/tools/)):
+To access the port `4000-4003` of the cluster, we can use the `kubectl port-forward`:
 
 ```
-kubectl port-forward svc/mycluster-frontend \
-  4000:4000 \
-  4001:4001 \
-  4002:4002 \
-  4003:4003
+kubectl -n default port-forward svc/mycluster-frontend  4000:4000 4001:4001 4002:4002 4003:4003
+```
+
+### Clean up
+
+If you want to terminate the GreptimeDB cluster, you can use the following command:
+
+```
+helm uninstall mycluster -n default
+helm uninstall etcd -n etcd-cluster
+helm uninstall greptimedb-operator -n greptimedb-admin
+```
+
+The CRDs of the greptimedb-operator are not deleted [by default](https://helm.sh/docs/topics/charts/#limitations-on-crds). You can delete them by the following command:
+
+```
+kubectl delete crds greptimedbclusters.greptime.io
+kubectl delete crds greptimedbstandalones.greptime.io
 ```
```

Comment:
And I think we can add a section about how to provide config files, it's a common scenario.

Reply:
It's already on my schedule.